scss/_detail.scss

```diff
@@ -1,7 +1,41 @@
+.container {
+  &-detail {
+    height: 100vh;
+    margin-top: 0;
+    position: relative;
+
+    &__wrapper {
+      align-items: center;
+      display: flex;
+      flex-direction: column;
+      height: 100%;
+      justify-content: center;
+      margin: 0 auto;
+      max-width: $desktop-width;
+    }
+  }
+}
+
+.back-button {
+  align-self: flex-start;
+  margin: 0 0 40px 40px;
+
+  &__link {
+    font-size: 20px;
+    &::before {
+      bottom: -2px;
+      content: url('C:\\Users/Julieta/Desktop/Personal/Wolox/Codigo/Intro-Maquetado/jm-angular/assets/back-arrow-20.png');
```

Comment:
For back arrow you can use the img tag, because it's a functional image not just for decorative purpose

Reply:
I use pseudo-element because Trello Card says that I have to use that 👐